### PR TITLE
update performance tool to run with .net 4.8

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

PerformanceTests do not run on .net 4.8 -
* The assembly bind redirects were doing very funny things to run the job on .net 4.7. This now runs with .net 4.8 is installed on my machine. The tests now run locally.

* package.config reference location is corrected.

I had nuget regenerate the assembly bind redirects and this is the much smaller app.config that it generated.

I have not yet actually tested this on the CI machine, will merge this first and address any failures after.

One thing I noticed is the test log will be much longer now since the logs are redirected to this log.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### CODE REVIEW

**Comment by mjkkirschner** [comment]
actually remembered I can run this:
https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoPerformance-CI/job/master/419/console

**Comment by QilongTang** [comment]
@mjkkirschner Did you run both the Core Perf tests and View ones?

File: tools/Performance/DynamoPerformanceTests/packages.config [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
+  <package id="System.Threading.Tasks.Parallel" version="4.3.0" targetFramework="net48" />
+  <package id="System.Threading.Thread" version="4.3.0" targetFramework="net48" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net48" />
+  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XmlDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XmlSerializer" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath.XDocument" version="4.3.0" targetFramework="net48" />
+  <package id="System.Xml.XPath.XmlDocument" version="4.3.0" targetFramework="net48" />
```
Comment (mmisol) [inline tools/Performance/DynamoPerformanceTests/packages.config:60]: Do we need to have all of these `System` packages? I don't think we have anything like that in our `Dynamo.All` solution after the upgrade.

**Comment by mjkkirschner** [comment]
@QilongTang I did not run the view perf tests as those are not part of the test framework on the CI. Maybe now that we have reduced our memory use significantly they will actually run without monotonically increasing in time.

File: tools/Performance/DynamoPerformanceTests/packages.config (hunk 1 above)
Comment (mjkkirschner) [reply tools/Performance/DynamoPerformanceTests/packages.config:60]: I think these are just direct dependencies of the version of benchmarkdotnet that we use - they are not actually copied to the bin folder of the perf executable, it seems they are just referenced from the GAC at runtime.

Comment (mmisol) [reply tools/Performance/DynamoPerformanceTests/packages.config:60]: Oh I see. The package targets .NET Standard 2.0 only.

I see dependencies for all .NET framework specific classes it uses, but also their dependencies recursively. I guess this is expected.

Feels like we should probably consider multi-targeting for our own NuGet packages in the future, as a bloated packages.config is not nice.

**Pull request merged**